rotkehlchen/tests/api/test_blockchain.py

```diff
@@ -784,29 +729,20 @@ def test_adding_editing_ens_account_works(rotkehlchen_api_server):
 
 
 @pytest.mark.parametrize('ethereum_accounts', [['0x9531C059098e3d194fF87FebB587aB07B30B1306']])
-def test_deleting_ens_account_works(rotkehlchen_api_server, ethereum_accounts):
+def test_deleting_ens_account_works(rotkehlchen_api_server):
     """Test that deleting an ENS eth account can be handled properly
 
     This test mocks all etherscan queries apart from the ENS ones
     """
     rotki = rotkehlchen_api_server.rest_api.rotkehlchen
-    setup = setup_balances(
-        rotki,
-        ethereum_accounts=ethereum_accounts,
-        btc_accounts=None,
-        eth_balances=None,
-        token_balances=None,
-    )
     request_data = {'accounts': ['rotki.eth']}
-    with ExitStack() as stack:
-        setup.enter_ethereum_patches(stack)
-        response = requests.delete(api_url_for(
-            rotkehlchen_api_server,
-            'blockchainsaccountsresource',
-            blockchain='ETH',
-        ), json=request_data)
-    result = assert_proper_response_with_result(response)
-    assert result['per_account'] == {}
+    response = requests.delete(api_url_for(
+        rotkehlchen_api_server,
+        "blockchainsaccountsresource",
```

Comment:
Please do not add `"`
```suggestion
        'blockchainsaccountsresource',
```